packages/contracts/contracts/MarginlyPool.sol

```diff
@@ -1260,7 +1260,9 @@ contract MarginlyPool is IMarginlyPool {
 
     uint256 realBaseCollateral = basePrice.mul(calcRealBaseCollateral(discountedBaseCollateral, discountedQuoteDebt));
     uint256 realQuoteDebt = quoteDebtCoeff.mul(discountedQuoteDebt);
-    systemLeverage.longX96 = uint128(Math.mulDiv(FP96.Q96, realBaseCollateral, realBaseCollateral.sub(realQuoteDebt)));
+    uint128 leverageX96 = uint128(Math.mulDiv(FP96.Q96, realBaseCollateral, realBaseCollateral.sub(realQuoteDebt)));
+    uint128 maxLeverageX96 = uint128(params.maxLeverage * FP96.Q96);
```

Comment:
Could you calculate it same `uint256 maxLeverageX96 = uint256(params.maxLeverage) << FP96.RESOLUTION;`